bin/cheribsdtest/cheribsdtest_vm.c

```diff
@@ -237,10 +255,7 @@ CHERIBSDTEST(vm_tag_shm_open_anon_shared2x,
 	cheribsdtest_success();
 }
 
-CHERIBSDTEST(vm_shm_open_anon_unix_surprise,
-    "test SHM_ANON vs SCM_RIGHTS",
-    .ct_xfail_reason =
-	"Tags currently survive cross-AS aliasing of SHM_ANON objects")
+CHERIBSDTEST(vm_shm_open_anon_unix_cross_as, "test SHM_ANON vs SCM_RIGHTS")
```

Comment:
That is probably a better name, yes.

Reply:
Along with the rename I've converted it into a positive test that revocation works so it's really a different test now.